nwb_conversion_tools/utils/common_writer_tools.py

```diff
@@ -158,11 +158,11 @@ def default_export_ops():
     return dict(
         use_times=False,
         write_as="raw",
-        es_key="ElectricalSeries",
+        es_key="ElectricalSeries_raw",
```

Comment:
I prefer "ElectricalSeries"

Reply:
Yes, its better to have the raw stored as "ElectricalSeries" only. I have kept the default to None instead. In the `write_electrical_series()` method, it defaults to "ElectricalSeries" if `es_key=None`. I think this is a better way to handle this as `es_key` will only be used if the user wishes to create some custom description for the `ElectricalSeries`. Also, if `es_key` is supplied, then its expected (else there is a warning [here line 413](https://github.com/catalystneuro/nwb-conversion-tools/pull/230/commits/ccb5886aef04a3ae3de6096394417746ad6cf1be#diff-abd8cf3d8732ccf67ab34bf977a3a7991ce438cba646d559d68ce174e69c1cf6R413) )then the user will also need to update the metadata and supply custom descriptions.